docs/source/execute_api.rst

```diff
@@ -0,0 +1,143 @@
+Executing notebooks
+===================
+
+.. module:: nbconvert.preprocessors
+
+In this section we show how to execute a ``.ipynb`` notebook
+document saving the result in notebook format.
+To export the notebook to other formats see section
```

Comment:
...notebook file to other formats, such as reStructured Text or Markdown, see ...